src/main/java/com/google/ehub/data/EntertainmentItem.java

```diff
@@ -15,15 +15,28 @@
   private final String title;
   private final String description;
   private final String imageUrl;
+  private final String releaseDate;
+  private final String runtime;
+  private final String genre;
+  private final String directors;
+  private final String writers;
+  private final String actors;
 
-  public EntertainmentItem(
-      Optional<Long> uniqueId, String title, String description, String imageUrl) {
+  private EntertainmentItem(Optional<Long> uniqueId, String title, String description,
+      String imageUrl, String releaseDate, String runtime, String genre, String directors,
+      String writers, String actors) {
```

Comment:
Fine for this PR, and honestly fine for this project. But I think as an API, this is a little weird because some of these fields you may not receive back from your API. I think I'd expect some of these to be optional as well and the ones that are absolutely necessary (e.g. title, imageUrl, maybe description?) would have checks in the constructor and throw IllegalArgumentException if not set.

Reply:
Yeah I agree this class is a little tricky because there can be a lot of possible values that may not always be fully provided by the items given in the API. Maybe I can find a better way to use these fields for Project Alpha and make the API cleaner.